packages/api/src/server.ts

```diff
@@ -157,6 +158,8 @@ const main = async (): Promise<void> => {
   // as healthy.
   await appDataSource.initialize()
 
+  await redisClient.connect()
```

Comment:
Maybe to make self hosters live easier we can make this optional, like `if env.redis { await redisClient.connect() }`

Reply:
hah, good timing. i just added redis to the docker compose. will revert the change 👍

Reply:
Yeah i mean ideally we can make redis optional for API, it doesn't look like that would be too hard since its mostly contained still

Reply:
yeah redis should be optional